java/src/main/java/com/powsybl/dataframe/network/adders/VscStationDataframeAdder.java

```diff
@@ -39,10 +39,7 @@ public class VscStationDataframeAdder extends AbstractSimpleAdder {
             SeriesMetadata.doubles("target_v"),
             SeriesMetadata.doubles("target_q"),
             SeriesMetadata.doubles("loss_factor"),
-            SeriesMetadata.booleans("voltage_regulator_on"),
-            SeriesMetadata.strings("bus_or_busbar_section_id"),
-            SeriesMetadata.ints("position_order"),
-            SeriesMetadata.strings("direction")
```

Comment:
just to be sure these metadata are in the new dataframes ?

Reply:
They are in ADDITIONNAL_METADATA in createFeederBay. It is only used for network modifications.